circle.yml

```diff
@@ -1,11 +1,15 @@
 machine:
-  services:
-    - docker
+  pre:
+    - sudo docker -d -e lxc -s btrfs -H tcp://0.0.0.0:2375:
+        background: true
+    - sleep 5
```

Comment:
Lol :sweat_smile: yeah.. giving time for docker to start :stuck_out_tongue_closed_eyes: